src/style.css

```
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  margin: 0 auto;
+}
+
+.card__description {
+  margin: 40px 0 0;
+  font-style: normal;
+  line-height: 18px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__code {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 0 0;
+}
+
+.card__rating {
+  margin: 16px 0 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__rating-reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star-active.svg");
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 > .stars__star:nth-child(1n + 5) {
+  background-image: url("images/star.svg");
+}
+
+.card__price {
+  margin: 24px 0 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__btn {
+  margin: 16px 0 0;
+  display: flex;
+  padding: 11.5px 55.5px 11.5px 54.5px;
+  background-color: #00acdc;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  border-radius: 5px;
+  color: #fff;
+  border: 1px solid #fff;
```

and on hover you can update border color

```suggestion
  border: 1px solid transparent;
```